rsync_backup.sh

```diff
@@ -179,26 +191,12 @@ MENU
         echo "$dest_is" "$dest_red"
         echo "$dest_is" "$dest_blue"
         echo
-
         check_if_directory "$source_path_Documents"
         check_if_directory "$dest_red"
         check_if_directory "$dest_blue"
-
```

Comment:
Similar to that last comment, you could also move these into `do_backup_for_2_targets`, so that each case would look like

```
echo "YOU CHOSE: ...
do_backup_for_2_targets ...
```

Reply:
Same comment as above.

Reply:
And this also looks good.